opslevel.yml

```diff
@@ -0,0 +1,12 @@
+version: 1
+service:
+  name: Heracles
+  description: Hiring challenge for Quality engineering positions
+  owner: quality
+  tags:
+    - key: type
+      value: hiring-challenge
```

Comment:
`other` sounds reasonable to me 👌

Reply:
@jsfr Will you please let us know when the change is implemented in OpsLevel side?

Reply:
oh no, please don't "other" .. that will quickly be used for all random kinds of things :( I don't know many challenges we currently maintain, probably worth a quick check and see if it's reasonable to go indeed with this?

Reply:
@steffkes Do you have a proposal for a key that this could go under that exists already or one with a better definition to add?

I get the concern with `other` but I'm also concerned with adding tons of "types", so in this case I would suggest we then at least try hard to think about what kind of high-level "thing" this is.

Reply:
Looking at [the repositories we have for challenges](https://github.com/orgs/pleo-io/repositories?q=challenge) (excluding the archived ones), we get 11 matches. While we currently have only **5** for `type=library`.

Makes me think it's probably fair enough to assign a separate type for that, just based on the usage

Reply:
👍 alright then, let's create a new type. @VictorPascualV you can go ahead and merge these and I will fix the check that looks for `type` in the meantime (it's a maturity check so nothing is preventing you from going ahead)

Reply:
I'll do that, thanks @jsfr 👍